src/borg_import/helpers/testsuite/test_timestamps.py

```diff
@@ -18,7 +18,14 @@ def test_datetime_from_mtime(tmpdir):
 
 
 def test_datetime_from_string():
-    assert datetime_from_string('1999-12-31T23:59:59') == datetime(1999, 12, 31, 23, 59, 59)
-    assert datetime_from_string('Mon Oct 31 23:35:50 UTC 2016') == datetime(2016, 10, 31, 23, 35, 50)
+    # Start with local timezone, convert to UTC, and then convert back to local.
+    assert datetime_from_string('1999-12-31T23:59:59') == datetime(1999, 12, 31, 23, 59, 59).astimezone()
```

Comment:
I somehow can't match the comment in 21 to the code in line 22.

Isn't it like that?:
lhs: a tz-aware timestamp in utc (coming from a string with date and time given in local timezone)
rhs: a tz-aware timestamp in local timezone (coming from a naive datetime object that is assumed to be in local timezone)

Reply:
If I understood your datetime_from_string change correctly, that function always returns a tz-aware datetime object that is always in utc timezone.

So, why does the test use a dt object in local tz on the rhs?

Reply:
Suggestion: change rhs to `datetime(1999, 12, 31, 23, 59, 59).astimezone(tz=timezone.utc)`

Reply:
Suggestion applied/comment removed.